.github/workflows/ci-release-gate.yml

```
@@ -38,180 +38,216 @@ jobs:
+      release_tag: ${{ needs.resolve.outputs.release_tag }}
+      github_sha: ${{ github.sha }}
+      platforms: linux/amd64
+    secrets:
+      dockerhub_username: ${{ vars.DOCKER_HUB_USERNAME }}
+      dockerhub_token: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      ghcr_username: ${{ github.repository_owner }}
+      ghcr_token: ${{ secrets.GITHUB_TOKEN }}
+  
+  generate-slsa-provenance-backend-ghcr:
+    needs: [build-push-backend]
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)